rubocop.yml

```
@@ -1,6 +1,7 @@
 AllCops:
   DisplayCopNames: true
   TargetRubyVersion: 2.6
+  SuggestExtensions: false
```

I would recommend disabling **NewCops** and only enable them after review and within a planned release of this gem. This way we're more flexible updating rubocop without breaking .workflows and such

```suggestion
  SuggestExtensions: false
  NewCops: disable
```